config.py

```diff
@@ -29,7 +38,11 @@ class Config(object):
     ASSET_PATH = STATIC_URL_PATH + '/'
     BASE_TEMPLATE_DATA = {
         'asset_path': ASSET_PATH,
-        'header_class': 'with-proposition'
+        'header_class': 'with-proposition',
+        "asset_fingerprints": {
+            "css": get_asset_fingerprint("app/static/stylesheets/application.css"),
```

Comment:
What if `application-ie6.css` changes but `application.css`? Looks like the fingerprint wouldn't change.